controls.py

```diff
@@ -0,0 +1,118 @@
+import pygame
+import serial
+import sympy
+import numpy
+import matplotlib
+import PySimpleGUI as sg
+import time
+
+class Control():
+    #Creates variables that store the motor labels for the different motors used. Some motor values have also been put in for adjustments that may be made to add the claw          
+    _CENTRAL_MOTOR = 1
+    _RIGHT_MOTOR = 2
+    _LEFT_MOTOR = 3
+
+    def __init__(self, scale, adjustment):
+        self.scale = scale 
+        self.adjustment = adjustment
+        self.running = False
+        self.list_movements = {
+            abutton: 0,
+            bbutton: 0,
+            xbutton: 0,
+            ybutton: 0,
+            left_motor_right: 0,
+            up_motor : 0,
+            right_motor_right: 0,
+            turning_amount: 0
+        }
+        
+        #Initializes the joystick being used, which is the XBox Controller
+        
+        self.controller = pygame.joystick.Joystick(0)
```

Comment:
It might not be a good idea to hard-code the controller here, as `pygame` might throw an error if the controller is not connected.

What you could instead do is have a list of `controllers` with `pygame.joystick.get_count()` (see Pygame documentation). Pygame events such as `pygame.CONTROLLERDEVICEADDED`/`pygame.JOYDEVICEADDED` and `pygame.CONTROLLERDEVICEREMOVED`/`pygame.JOYDEVICEREMOVED` can also be used.

Reply:
Does using the if statement work, or do I have to use the events?